modules/ROOT/attachments/schema-registry-api.json

```
@@ -441,6 +441,13 @@
             "in": "path",
             "required": true,
             "type": "integer"
+          },
+          {
+            "name": "format",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "For Avro and Protobuf schemas only (Redpanda 25.2 or later). Supported values: an empty string `''` returns the schema in the current format (default), and `serialized` (Protobuf only) returns the schema in its wire binary format in Base64. Unsupported values return a 501 error."
```

```suggestion
            description: In Redpanda 25.2 or later, available for Avro and Protobuf schemas. Accepted values: an empty string `''` returns the schema in its current format (default), and `serialized` (Protobuf only) returns the schema in Base64-encoded wire binary format. Unsupported values return a 501 error.
```

I would use the version requirement as the first thing. And in other docs we use "accepted values", so suggesting this for consistency.

---

Updated